src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,19 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine truck = new Truck(1);
+        Machine buldozer = new Bulldozer(1);
+        Machine excavator = new Excavator(1);
 
+        Machine[] machines = new Machine[3];
+        machines[0] = truck;
+        machines[1] = buldozer;
+        machines[2] = excavator;
```

Comment:
you can do it in one line like this
`Object[] objects = {new Object(), new Object(), new Object(), new Object(), new Object()};`